recipe/bld.bat

```diff
@@ -6,22 +6,32 @@ if errorlevel 1 exit 1
 set MENU_DIR=%PREFIX%\Menu
 IF NOT EXIST (%MENU_DIR%) mkdir %MENU_DIR%
 
+rem  Copy Spyder's icon
 copy %RECIPE_DIR%\spyder.ico %MENU_DIR%\spyder.ico
 
+rem  Replace variables in menu files
 for /F "delims=. tokens=1" %%i in ("%PKG_VERSION%") do set PKG_MAJOR_VER=%%i
+call :replace spyder-menu.json spyder-menu.json
+call :replace spyder-menu-v1.json spyder-menu-v1.json.bak
 
-for /f "delims=" %%i in (%RECIPE_DIR%\spyder-menu.json) do (
-    set s=%%i
-    set s=!s:__PKG_VERSION__=%PKG_VERSION%!
-    echo !s:__PKG_MAJOR_VER__=%PKG_MAJOR_VER%!>> %MENU_DIR%\spyder-menu.json
+rem  Copy GUI executable stub
+for /F "tokens=*" %%i in (
+    '%CONDA_PYTHON_EXE% -c "import conda_build, pathlib; print(pathlib.Path(conda_build.__file__).parent / 'gui-64.exe')"'
+) do (
+    set exe_path=%%i
 )
+copy /y /b %exe_path% %SCRIPTS%
 
-for /f "delims=" %%i in (%RECIPE_DIR%\spyder-menu-v1.json) do (
-    set s=%%i
-    set s=!s:__PKG_VERSION__=%PKG_VERSION%!
-    echo !s:__PKG_MAJOR_VER__=%PKG_MAJOR_VER%!>> %MENU_DIR%\spyder-menu-v1.json.bak
-)
+rem  Copy launch script
+copy /y %RECIPE_DIR%\spyder-script.pyw %SCRIPTS%
+
+:exit
+    exit /b %errorlevel%
 
-del %SCRIPTS%\spyder_win_post_install.py
-del %SCRIPTS%\spyder.bat
-del %SCRIPTS%\spyder
```

Comment:
Why are not we removing these scripts now? I think they shouldn't be part of the package.

Reply:
I agree that they should not be part of the package. I removed these lines because these files were not being added to the `%SCRIPTS%` directory, so I assumed they were tech-debt. However, I can double check.

Reply:
Yes, I've confirmed that these files are not in the Scripts directory of the conda package (or anywhere else). Only the following:
```
Scripts/.spyder-post-link.bat
Scripts/gui-64.exe
Scripts/spyder-script.py
Scripts/spyder-script.pyw
Scripts/spyder.exe
Scripts/spyder.ico
```